Gemfile.lock

```diff
@@ -161,7 +169,7 @@ GEM
     case_transform (0.2)
       activesupport
     charlock_holmes (0.7.6)
-    chewy (5.0.1)
+    chewy (5.1.0)
```

Comment:
https://github.com/toptal/chewy/commit/7c5096d40ea87463b00c047b0184af7956f1b31d

chewy 5.0.1 has been canceled and is now 5.1.0.